model.py

```diff
@@ -615,6 +617,32 @@ def update(self, start, end, position):
         UniqueConstraint('patron_id', 'license_pool_id'),
     )
 
+class Annotation(Base):
+    LS_NAMESPACE = u"http://librarysimplified.org/terms/annotation/"
+
+    IDLING = LS_NAMESPACE + u'idling'
+
+    MOTIVATIONS = [
+        IDLING
+    ]
+
+    __tablename__ = 'annotations'
+    id = Column(Integer, primary_key=True)
+    patron_id = Column(Integer, ForeignKey('patrons.id'), index=True)
+    identifier_id = Column(Integer, ForeignKey('identifiers.id'), index=True)
+    motivation = Column(
+        Enum(*MOTIVATIONS, name="motivation"),
+        index=True
+    )
+    timestamp = Column(DateTime, index=True)
+    active = Column(Boolean, default=True)
+    content = Column(Unicode)
+    target = Column(Unicode)
```

Comment:
Oh, yeah, it doesn't even need to be a hash. An ETag can be a timestamp.

However we should preface it with `W/` (e.g. `ETag: W/"2016-08-01:01:04:11"`) to make it a weak validator since we can't guarantee a new version is byte-for-byte identical with an old version. (Not that the content is likely to change without a timestamp update, but the way we generate the JSON-LD might change.)